src/pykiso/cli.py

```diff
@@ -209,6 +126,12 @@ def eval_user_tags(click_context: click.Context) -> Dict[str, List[str]]:
     is_flag=True,
     help="stop the test run on the first error or failure",
 )
+@click.option(
+    "--verbose",
```

Comment:
for verbose it is very common to have a short form.
So just add    "  -v  "

Reply:
also update the docs then (:

Reply:
Short form added